src/Core/Form/IdentifiableObject/DataProvider/CategoryFormDataProvider.php

```diff
@@ -88,6 +95,7 @@ public function getDefaultData()
         $defaultGroups = $this->queryBus->handle(new GetDefaultGroups());
```

Comment:
@matks we have 2 ways of getting default groups:

1. By using Query like above.
2. By using `\PrestaShop\PrestaShop\Adapter\Group\Provider\DefaultGroupsProvider` service

What would be the correct way? :thinking: we have to get rid of one of these 2.

Reply:
A bit hard to answer because this is the borderline of CQRS. I think since this is kind of configuration we can go for solution 2, using DefaultGroupsProvider. But please add a `@todo` statement on it saying something like "Not sure whether or not to use a Query here". This way later we'll know where to look at, and we might be able to validate this choice using the knowledge and experience acquired in the meantime.